### PR TITLE
move transfer/style mobile css to appstyles

### PR DESCRIPTION
**Pull request by noodlemoodle**
move mobile styles for some foam views from transfer/Style to AppStyles
nano pr: https://github.com/nanoPayinc/NANOPAY/pull/13684/files

### CODE REVIEW

File: src/foam/nanos/controller/AppStyles.js [hunk 1]
```diff
@@ -272,11 +272,53 @@ foam.CLASS({
     font-weight: bold;
     font-size: 2.8rem;
   }
-  @media only screen and (min-width: 321px) {
+
+  /* mobile */
+  @media only screen and (min-width: 321px) and (max-width: 767px) {
```
Comment (sarthak-marwaha) [inline src/foam/nanos/controller/AppStyles.js]: Don't need `max-width`, here and elsewhere

**Review by zahnno — APPROVED** [review]

File: src/foam/nanos/controller/AppStyles.js (hunk 1 above)
Comment (noodlemoodle) [reply src/foam/nanos/controller/AppStyles.js]: changed to select max-width instead since min-width will also override styles for desktop sizes

**Comment by jlhughes** [comment]
@noodlemoodle @sarthak-marwaha a lot of ! important in here. I know you were working to reduce those elsewhere.

**Comment by sarthak-marwaha** [comment]
> @noodlemoodle @sarthak-marwaha a lot of ! important in here. I know you were working to reduce those elsewhere.

@jlhughes Talked to @noodlemoodle about it on slack, reason for it is that these styles should be in the components and mobile CSS should be default. Expecting further updates to the this PR to fix that

**Pull request merged**